src/Sarif/Autogenerated/SarifLogEqualityComparer.cs

```diff
@@ -84,7 +84,7 @@ public int GetHashCode(SarifLog obj)
                         result = result * 31;
                         if (value_0 != null)
                         {
-                            result = (result * 31) + value_0.GetHashCode();
+                            result = (result * 31) + value_0.ValueGetHashCode();
```

Comment:
Here's another example, and this one's a bug.

This is why I'm so adamant about not letting files linger in `NotYetAutogenerated`. It's not just debt, it can cause bugs, and it's just too hard to make sure it's kept up to date.